apps/approval/appconfig.py

```diff
@@ -4,3 +4,7 @@
 class ApprovalConfig(AppConfig):
     name = 'apps.approval'
     verbose_name = 'Approval'
+
+    def ready(self):
+        super().ready()
+        import apps.approval.signals  # flake8: noqa
```

Comment:
Cool, then flake8 uses the comment syntax for type hinting. Guido and Co just released another PEP with actual type hinting syntax based on reserved syntax that will replace the comment syntax. Just thought it was interesting that they used a language feature that was not in standard use.